### ISSUE TITLE
Is this great project in active development or it's abandoned?

### ISSUE THREAD

**Issue by mikekgr** [subhra74/snowflake#220]
Please @subhra74 give us your answer.
Many thanks!

**Comment by rzel** [comment]
I have the same question. This is by far the greatest tool for the job I have ever come across. If the author needs donations to keep going, I would gladly help out and I'm sure others would too.

**Comment by mikekgr** [comment]
The only hope is the answer (if any) from the developer...

**Comment by GitHunter0** [comment]
I agree. I can't understand why the owner disappeared, Muon is the best tool so far. 
Probably is a good idea to follow this fork here: https://github.com/devlinx9/muon-ssh

**Comment by mikekgr** [comment]
Thanks for let us know the fork !
